scripts/requirements.txt

```diff
@@ -20,4 +20,6 @@ pymongo>=3.7.2
 pytest>=7.1.2
 scipy>=1.7.3
 statsmodels>=0.13.2
-gnssanalysis>=0.0.28
+#georinex>=1.16.2
+git+https://github.com/breid-phys/georinex.git # fast processing patch - not yet merged into georinex https://github.com/geospace-code/georinex/pull/85
+-e . # Add local gn
```

Comment:
Is this here as a convenience primarily?
I gather it's streamlining things for a better user experience (same effect as running `pip install .` after gathering the above requirements?)

Reply:
Yep just for convenience

Reply:
After some other discussions I realised `-e` apparently _symlinks_, not installs. Still convenient, but maybe a bit of unexpected behaviour we should avoid doing in `requirements.txt`?

Reply:
Agree, see commit [998bc72](https://github.com/GeoscienceAustralia/ginan/pull/62/commits/998bc72b9fac517f391f52fd7fe63c2e3bb5cc46) which removes it from `requirements.txt` and adds a new instruction to the README to do a `pip install .`